platform/admin/core/admin-core-insecuredefaults/src/main/java/org/codice/ddf/admin/insecure/defaults/service/InsecureDefaultsCollector.java

```diff
@@ -244,4 +273,37 @@ private String getTruststorePassword() {
   List<Validator> getValidators() {
     return validators;
   }
+
+  SystemNotice createUsersPropertiesNotice() {
+    UserPropertiesDeletionScheduler scheduler = new UserPropertiesDeletionScheduler();
+    Date firstInstall = scheduler.installationDate();
+    Set<String> details = new HashSet<>();
+    String title;
+
+    if (firstInstall != null) {
+      Date timestamp = new Date(firstInstall.getTime() + TimeUnit.DAYS.toMillis(3));
+
+      title =
+          String.format(
+              "USERS.PROPERTIES FILE FOUND! Follow the instructions below before %s to prevent the system from locking down.",
+              timestamp.toString());
+      details.add(
+          String.format(
+              "A user.properties file was found at [%s]. It will be automatically deleted on %s. After the deletion of this file you will not be able to log back in to the system. Please follow the hardening guides in the documentation under Removing Default Users to avoid the system lock down.",
+              USERS_PROPERTIES_FILE, timestamp.toString()));
+
+      details.addAll(
+          userPropertiesAlerts.stream().map(Alert::getMessage).collect(Collectors.toSet()));
```

Comment:
❓ Are we guaranteed that userPropertiesAlerts will always be non-null? We might want to set the default value to an empty list or such when this variable is initialized to prevent that